Fmaraev/Hometask_1/Task_1.py

```diff
@@ -0,0 +1,58 @@
+import random
+print('Hello, world!')
+name = input('What is your name?\n')
+print('Hi, %s.' % name)
+if name == 'Федя':
+    friends = ['Алексей', 'Андрей', 'Артем', 'Михаил']
+    for i, name in enumerate(friends):
+        print("Дргу номер {iteration} это {name}"
+              .format(iteration=i, name=name))
+"""Пример работы простой функции"""
+
+
+def greet(name_2):
+    print('Hello', name_2)
+
+
+greet('Ололо')
+greet('Провайдер')
+greet('Участник')
+
```

Comment:
It would be nice to put two lines between these calls and docstring and one line after docstring